packages/lib/src/vis/heatmap/HeatmapMesh.tsx

```diff
@@ -123,6 +89,7 @@ function HeatmapMesh(props: Props) {
       min: { value: scaledDomain[0] },
       oneOverRange: { value: 1 / (scaledDomain[1] - scaledDomain[0]) },
       gammaExponent: { value: gammaExponent },
+      normRevertFactor: { value: values.dtype === 'uint8' ? 255 : 1 }, // to revert automatic normalization
```

Comment:
It might be useful to specify what does the normalization in the first place (is it three.js with a particular texture format ?)